2020-08-18 12:44:25,340	INFO resource_spec.py:231 -- Starting Ray with 158.06 GiB memory available for workers and up to 71.74 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
2020-08-18 12:44:26,556	INFO services.py:1193 -- View the Ray dashboard at [1m[32mlocalhost:8265[39m[22m
2020-08-18 12:44:27,707	INFO trainer.py:605 -- Tip: set framework=tfe or the --eager flag to enable TensorFlow eager execution
2020-08-18 12:44:27,708	INFO trainer.py:632 -- Current log_level is WARN. For more information, set 'log_level': 'INFO' / 'DEBUG' or use the -v and -vv flags.
WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0306438>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e03069b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e3c88>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e33c8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0306438>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e03069b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e3c88>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e33c8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0306438>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e03069b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e3c88>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e02e33c8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06852b0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f06857f0>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02f05e6668>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f02e0337dd8>> and will run it as-is.
Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31083)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31083)[0m Instructions for updating:
[2m[36m(pid=31083)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31080)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31080)[0m Instructions for updating:
[2m[36m(pid=31080)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31064)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31064)[0m Instructions for updating:
[2m[36m(pid=31064)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31055)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31055)[0m Instructions for updating:
[2m[36m(pid=31055)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31059)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31059)[0m Instructions for updating:
[2m[36m(pid=31059)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31072)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31072)[0m Instructions for updating:
[2m[36m(pid=31072)[0m non-resource variables are not supported in the long term
[2m[36m(pid=31067)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/compat/v2_compat.py:88: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
[2m[36m(pid=31067)[0m Instructions for updating:
[2m[36m(pid=31067)[0m non-resource variables are not supported in the long term
2020-08-18 12:46:52,343	INFO trainable.py:251 -- Trainable.setup took 144.659 seconds. If your trainable is slow to initialize, consider setting reuse_actors=True to reduce actor creation overheads.
2020-08-18 12:46:52,343	WARNING util.py:37 -- Install gputil for GPU system monitoring.
Exp dir detected: /home/ubuntu/ray_results/APEX_PongNoFrameskip-v4_2020-08-18_12-44-27xita8hqi
[2m[36m(pid=31080)[0m 2020-08-18 12:46:52.456734: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2m[36m(pid=31080)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
[2m[36m(pid=31080)[0m Instructions for updating:
[2m[36m(pid=31080)[0m If using Keras pass *_constraint arguments to layers.
[2m[36m(pid=31055)[0m 2020-08-18 12:46:52.462945: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2m[36m(pid=31055)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
[2m[36m(pid=31055)[0m Instructions for updating:
[2m[36m(pid=31055)[0m If using Keras pass *_constraint arguments to layers.
[2m[36m(pid=31067)[0m 2020-08-18 12:46:52.532389: E tensorflow/stream_executor/cuda/cuda_driver.cc:351] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
[2m[36m(pid=31067)[0m WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/tensorflow_core/python/ops/resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
[2m[36m(pid=31067)[0m Instructions for updating:
[2m[36m(pid=31067)[0m If using Keras pass *_constraint arguments to layers.
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0630>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8630>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102630>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0b38>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8b38>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102b38>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649940>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743940>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061940>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649dd8>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743dd8>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061dd8>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0630>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0b38>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649940>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8630>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8b38>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102630>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649dd8>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743940>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743dd8>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102b38>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061940>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061dd8>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0630>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c06f0b38>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8630>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102630>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b07e8b38>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649940>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c0649dd8>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743940>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b0743dd8>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c102b38>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061940>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c061dd8>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ff8d0>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9e00ffe10>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131bc50>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855f8d0>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b855fe10>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c2838d0>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c283e10>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31080)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fb9c131be80>> and will run it as-is.
[2m[36m(pid=31080)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31080)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fc50>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31055)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7fd5b843fe80>> and will run it as-is.
[2m[36m(pid=31055)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31055)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163c50>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
[2m[36m(pid=31067)[0m WARNING:tensorflow:AutoGraph could not transform <bound method NoisyLayer.call of <ray.rllib.models.tf.layers.noisy_layer.NoisyLayer object at 0x7f586c163e80>> and will run it as-is.
[2m[36m(pid=31067)[0m Please report this to the TensorFlow team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output.
[2m[36m(pid=31067)[0m Cause: unexpected indent (<unknown>, line 17)
Traceback (most recent call last):
  File "train.py", line 83, in <module>
    result = trainer.train()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/agents/trainer.py", line 519, in train
    raise e
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/agents/trainer.py", line 508, in train
    result = Trainable.train(self)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/tune/trainable.py", line 332, in train
    result = self.step()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/agents/trainer_template.py", line 110, in step
    res = next(self.train_exec_impl)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 758, in __next__
    return next(self.built_iterator)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 845, in apply_filter
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 845, in apply_filter
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 845, in apply_filter
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 1078, in build_union
    item = next(it)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 758, in __next__
    return next(self.built_iterator)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 785, in apply_foreach
    for item in it:
  [Previous line repeated 2 more times]
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 552, in base_iterator
    batch = ray.get(obj_ref)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/worker.py", line 1538, in get
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError(DependencyNotInstalled): [36mray::RolloutWorker.par_iter_next_batch()[39m (pid=31055, ip=172.31.80.116)
  File "python/ray/_raylet.pyx", line 479, in ray._raylet.execute_task
  File "python/ray/_raylet.pyx", line 432, in ray._raylet.execute_task.function_executor
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 1162, in par_iter_next_batch
    batch.append(self.par_iter_next())
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/util/iter.py", line 1156, in par_iter_next
    return next(self.local_it)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/evaluation/rollout_worker.py", line 281, in gen_rollouts
    yield self.sample()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/evaluation/rollout_worker.py", line 563, in sample
    batches = [self.input_reader.next()]
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/evaluation/sampler.py", line 71, in next
    batches = [self.get_data()]
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/evaluation/sampler.py", line 181, in get_data
    item = next(self.rollout_provider)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/evaluation/sampler.py", line 505, in _env_runner
    base_env.poll()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/env/base_env.py", line 325, in poll
    self.new_obs = self.vector_env.vector_reset()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/env/vector_env.py", line 133, in vector_reset
    return [e.reset() for e in self.envs]
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/ray/rllib/env/vector_env.py", line 133, in <listcomp>
    return [e.reset() for e in self.envs]
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitor.py", line 39, in reset
    self._after_reset(observation)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitor.py", line 185, in _after_reset
    self.reset_video_recorder()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitor.py", line 206, in reset_video_recorder
    self.video_recorder.capture_frame()
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitoring/video_recorder.py", line 116, in capture_frame
    self._encode_image_frame(frame)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitoring/video_recorder.py", line 162, in _encode_image_frame
    self.encoder = ImageEncoder(self.path, frame.shape, self.frames_per_sec, self.output_frames_per_sec)
  File "/home/ubuntu/anaconda3/envs/tensorflow2_p36/lib/python3.6/site-packages/gym/wrappers/monitoring/video_recorder.py", line 255, in __init__
    raise error.DependencyNotInstalled("""Found neither the ffmpeg nor avconv executables. On OS X, you can install ffmpeg via `brew install ffmpeg`. On most Ubuntu variants, `sudo apt-get install ffmpeg` should do it. On Ubuntu 14.04, however, you'll need to install avconv with `sudo apt-get install libav-tools`.""")
gym.error.DependencyNotInstalled: Found neither the ffmpeg nor avconv executables. On OS X, you can install ffmpeg via `brew install ffmpeg`. On most Ubuntu variants, `sudo apt-get install ffmpeg` should do it. On Ubuntu 14.04, however, you'll need to install avconv with `sudo apt-get install libav-tools`.
